content/departments/tech-ops/tools/drive.md

```diff
@@ -0,0 +1,35 @@
+# Google Drive
+
+We use Google Drive to store documents and collaborate internally and externally. All files should be stored within Google Drive and not on your local hard drive.
+
+## Sharing permissions
+
+There are several ways to share documents within Google Drive.
+
+- **Restricted**: Only people specifically invited to the file can access that file. Collaborators can include anyone with an @sourcegraph email address or a user from an allowlisted domain. This setting is standard to all Sourcegraph accounts.
+- **Sourcegraph**: Anyone with an @sourcegraph email can access the document with the link to the file. Collaborators can include anyone with an @sourcegraph email address or any user from an allowlisted domain. This setting is standard to all Sourcegraph accounts.
+- **Anyone with the link** : Anyone with the link can access the document. This should be restricted to only files that can be shared publicly and do not contain any sensitive teammate or client information. This permission is limited to our publicly shared folders only.
+
+From each of these sharing levels you can set how you want collaborates to interact with the file (edit, view, or comment). More information about getting started with Google Drive can be found [here.](https://support.google.com/a/users/answer/9310248?hl=en)
```

Comment:
I think there's a typo here --> "...you can set how you want collaborators to interact..."